lisa/node.py

```diff
@@ -157,8 +156,13 @@ def _check_sudo_available(self) -> bool:
         process = self._execute("ls", shell=True, sudo=True, no_info_log=True)
         result = process.wait_result(10)
         if result.exit_code != 0:
-            self.log.debug("node doesn't support sudo /bin/sh.")
-            return False
+            self.log.debug(
+                f"_check_sudo_available: 'sudo /bin/sh ls' not succeeded,stderr: {result.stderr}"
```

Comment:
It looks this line is not necessary. The command log outputs the raw output already. And when it runs remotely, the stderr is empty. Please read the generated log to make sure no duplicate logs, and the content output is expected.